SDK/AppCenterAuth/Microsoft.AppCenter.Auth.iOS.Bindings/Microsoft.AppCenter.Auth.iOS.Bindings.csproj

```diff
@@ -59,7 +59,7 @@
       <ForceLoad>True</ForceLoad>
       <WeakFrameworks>AuthenticationServices</WeakFrameworks>
       <SmartLink>False</SmartLink>
-      <Frameworks>WebKit SafariServices</Frameworks>
+      <Frameworks>WebKit SafariServices CoreTelephony SystemConfiguration UserNotifications</Frameworks>
```

Comment:
CoreTelephony SystemConfiguration should be in Core
UserNotifications should be weak and in Push

Reply:
Source: https://github.com/microsoft/appcenter-sdk-apple/blob/master/AppCenter.podspec